File: UNIT 07/SOURCE CODE/CSHARP/UnOrderedArrayList/UnOrderedArrayList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UnOrderedArrayList
{
    public class UnOrderedArrayList
    {
        


        public static void Main(string[] args)
        {

            List L = new List(); //declare and initialize list L
           

            L.Insert(10); L.Insert(15); L.Insert(20);
            L.ShowData();
            L.Delete(15);
            L.ShowData();
            L.Insert(30); L.Insert(50);L.Delete(10);
            L.ShowData();

            Console.ReadKey();
        }
    }
}

File: UNIT 07/SOURCE CODE/CPLUSPLUS/UnOrderedArrayList/UnOrderedArrayList.cpp
// UnOrderedArrayList.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include "pch.h"
#include <iostream>
#include "List.h"

int main()
{
	List L;
	Init(L);
	Insert(10 , L); Insert(15, L); Insert(20, L);
	ShowData(L);
	Delete(15, L);
	ShowData(L);
	Insert(30, L); Insert(50, L); Delete(10, L);
	ShowData(L);
	system("pause");
}


